CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.25.0
+*This version supports all current LTS versions of Node.js (10.x, 12.x, 14.x, 16.x). Older versions might still work but are untested from now on!*
+
+* (everhardt/JKRhb) feat: add clientIdentifier option to createServer
+* (JKRhb) feat: add more content-formats
+* (ats-org/Jamezo97/invaderb) feat: Type-1 Block-wise transfer
+* (sjlongland/pekkanikander ) feat: Add support for de-registering observations
```

Comment:
@Apollon77 Thank you for composing the changelog! I only noticed this trailing whitespace. Other than that, LGTM :)

```suggestion
* (sjlongland/pekkanikander) feat: Add support for de-registering observations
```